rust/core-lib/src/tabs.rs

```diff
@@ -319,6 +319,7 @@ impl CoreState {
             // handled at the top level
             ClientStarted { .. } => (),
             SetLanguage { view_id, language_id } => self.do_set_language(view_id, language_id),
+            ToggleTail { enabled } => self.file_manager.tail_toggle = enabled,
```

Comment:
I think we want this to be a per-view option; so it should include a `ViewIdentifier`, and should apply to that particular view, not to all files.

Reply:
This is something I wasnt sure about when I implemented. I was going to implement per view tail but then thought if a file isn't active, no events should be triggered, including `Write` event. I need to handle a case where an inactive tab becomes active.

Reply:
we receive events regardless of whether we are focused on a view. 👍

Reply:
Implemented per-view option. Client side changes are pending.